spring-boot-data-dynamodb/src/main/resources/templates/index.html

```diff
@@ -0,0 +1,19 @@
+<html>
+<head>
+	<title>WebJars Demo</title>
+	<link rel="stylesheet" href="/webjars/bootstrap/3.3.4/css/bootstrap.min.css" />
+</head>
+<body>
+
+	<div class="container"><br/>
+		<div class="alert alert-success">
+		  <a href="#" class="close" data-dismiss="alert" aria-label="close">�</a>
+		  <strong>Success!</strong> It is working as we expected.
+		</div>
+	</div>
+
+<script src="/webjars/jquery/2.1.4/jquery.min.js"></script>
+<script src="/webjars/bootstrap/3.3.4/js/bootstrap.min.js"></script>
+
+</body>
+</html>
```

Comment:
Also this project was copied from an existing accepted project and was not changed @pkhanal